README.md

```diff
@@ -5,85 +5,51 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/mattermost/mattermost-plugin-circleci)](https://goreportcard.com/report/github.com/mattermost/mattermost-plugin-circleci)
 [![Mattermost Community Channel](https://img.shields.io/badge/Mattermost%20Community-~Plugin%3A%20CircleCI-blue)](https://community.mattermost.com/core/channels/plugin-circleci)
```

Comment:
I really like the HW badge like the one in https://github.com/Reene-Simon/mattermost-plugin-splunk/blob/93458f873554e49104de037d09429d30449ef3cc/README.md?

2/5 I would like to see it here as well.

Reply:
![image](https://github.com/mattermost/mattermost-plugin-circleci/assets/102587100/d86cc1fc-bbe5-4c8b-82a6-3b2cae6dd627)

@cwarnermm sure i shall work on it, could you just confirm once that is this what I should include?

Reply:
Yes - confirmed. Thanks, @heisdinesh!

Reply:
@cwarnermm @hanzei I have added the badge